xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd

```diff
@@ -177,6 +178,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="water"/>
 			<xsd:enumeration value="cableway"/>
 			<xsd:enumeration value="funicular"/>
+			<xsd:enumeration value="SnowAndIce"/>
```

Comment:
What is the reason that this is not "snowAndIce" (lower S)

Reply:
![image](https://user-images.githubusercontent.com/502394/117212332-7fa6d000-adfa-11eb-8e2b-c5548705fe99.png)